src/core/init/utils/initial_seek_and_play.ts

```diff
@@ -127,7 +127,47 @@ export default function performInitialSeekAndPlay(
     // a sufficient `readyState` has been reached for directfile contents.
     // So let's divide the two possibilities here.
     const initialTime = typeof startTime === "function" ? startTime() : startTime;
-    if (shouldPreventSeekingAt0Initially() && initialTime === 0) {
+    if (
+      initialTime === undefined &&
+      isDirectfile &&
+      mediaElement.readyState < HTMLMediaElement.HAVE_CURRENT_DATA
+    ) {
+      /**
+       * On browser, such as Safari, the HTMLMediaElement.duration
+       * and HTMLMediaElement.buffered may not be initialized at readyState 1, leading
+       * to cases where it can be equal to `Infinity`.
+       * If so, the range in which it is possible to seek is not yet known.
```

Comment:
It's more in reality about: the starting position may not be known yet

Reply:
I have changed the comment, does it look more representative of what it does ?

Reply:
If so I might also change that comment for the V4

Reply:
> I have changed the comment, does it look more representative of what it does ?

Yes thanks